grafana-plugin/src/pages/integrations/Integrations.tsx

```diff
@@ -458,14 +471,16 @@ class _IntegrationsPage extends React.Component<IntegrationsProps, IntegrationsS
         )}
       </div>
     );
-  }
+  };
 
-  renderMaintenance(item: ApiSchemas['AlertReceiveChannel']) {
+  renderMaintenance = (item: ApiSchemas['AlertReceiveChannel']) => {
+    const { theme } = this.props;
     const maintenanceMode = item.maintenance_mode;
+    const utilStyles = getUtilStyles(theme);
 
     if (maintenanceMode === MaintenanceMode.Debug || maintenanceMode === MaintenanceMode.Maintenance) {
       return (
-        <div className={cx('u-flex')}>
+        <div className={cx(utilStyles.flex)}>
```

Comment:
```suggestion
        <div className={utilStyles.flex}>
```

Reply:
Good point, I need to apply regex for these cases too! :+1: